docs/guide/controllers.md

```diff
@@ -172,50 +172,139 @@ The Query controller makes it easy to retrieve content without writing custom PH
+```
+
+#### RelatedToContent
+
+The `RelatedToContent` Query Type retrieves content that is a reverse relation to the provided Content item.
+
+It takes `content` or `field` as obligatory parameters.
+`field` indicates the Relation or RelationList Field that contains the relations.
+
+``` yaml
+params:
+    query:
+        query_type: 'RelatedToContent'
+        parameters:
+            content: '@=content'
+            field: 'relations'
+        assign_results_to: items
+```
+
+#### GeoLocation
+
+The `GeoLocation` Query Type retrieves content by distance of the location provided in a MapLocation Field.
+
+It takes the following obligatory parameters:
+
+- `field` - MapLocation Field identifier
+- `distance` - distance to check for
+- `latitude` and `longitude` - coordinates of the location to check distance to
+- (optional) `operator` - operator to check value against, by default `<=`
+
+``` yaml
+params:
+    query:
+        query_type: 'GeoLocation'
+        parameters:
+            field: 'location'
+            distance: 200
+            latitude: '@=content.getFieldValue("location").latitude'
+            longitude: '@=content.getFieldValue("location").longitude'
+            operator: '<'
+        assign_results_to: items
 ```
 
+#### General Query Type parameters
+
+All built-in Query Types take the following optional parameters:
+
+- `limit`	- limit for number of search hits to return
+- `offset` - offset for search hits, used for paging the results
+- `sort` - sorting options
+- `filter` - additional query filters:
+    - `content_type` - return only results of given content types
+    - `visible_only` - return only visible results (default `true`)
+    - `siteaccess_aware` - return only results limited to current SiteAccess root (default `true`)
+
+For example:
+
+``` yaml
+params:
+    query:
+        query_type: 'Children'
+        parameters:
+            content: '@=content'
+            filter:
+                content_type: ['blog_post']
+                visible_only: false
+            limit: 5
+            offset: 2
+            sort: 'content_name asc, date_published desc'
+        assign_results_to: items
+```
+
+!!! note "Sorting order"
+
+    To provide sorting order to the `sort` parameter, use names of the Sort Clauses in
+    `\eZ\Publish\API\Repository\Values\Content\Query\SortClause`.
+
+#### Using built-in Query Types
+
+This example assumes a "Blog" container that contains a set of "Blog post" items. The goal is, when viewing a Blog, to list the Blog posts it contains.
+
+Two items are required:
+
+- a View template - It will render the Blog, and list the Blog posts it contains
+- a `content_view` configuration - It will instruct Platform, when viewing a Content item of type Blog, to use the Query Controller, the view template, and the `Children` QueryType. It will also map the id of the viewed Blog to the QueryType parameters, and set which Twig variable the results will be assigned to.
```

Comment:
```suggestion
- a `content_view` configuration - It will instruct Platform, when viewing a Content item of type Blog, to use the Query Controller, the view template, and the `Children` QueryType. It will also map the ID of the viewed Blog to the QueryType parameters, and set which Twig variable the results will be assigned to.
```